package.json

```diff
@@ -1,21 +1,20 @@
 {
   "name": "business-filings-ui",
-  "version": "6.6.8",
+  "version": "6.7.0",
   "private": true,
   "appName": "Filings UI",
   "sbcName": "SBC Common Components",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
-    "lint:nofix": "vue-cli-service lint --no-fix",
-    "test:e2e": "vue-cli-service test:e2e",
-    "test:unit": "vue-cli-service test:unit --testPathPattern --coverage",
-    "test:unit-silent": "vue-cli-service test:unit --silent",
-    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "lint": "eslint . --ext .js,.ts,.vue",
+    "test:unit": "vitest run --silent",
```

Comment:
This scares me a bit -- how silent are the tests now? Do we still see test errors and console output?

However, I think this might be a good option to add to the test:coverage script (since that command is concerned with coverage, not errors).

Reply:
Console output:
![console output initial](https://github.com/bcgov/business-filings-ui/assets/122301442/8a369619-f2c2-4c0f-b071-46a753c4d3e8)
![console output end](https://github.com/bcgov/business-filings-ui/assets/122301442/0dd122a5-9ed1-49e0-8f59-de8d094c6864)

You're right. So, how about I revert the change here and add a new script:
`"test:coverage": "vitest run --silent"`
Thoughts?

Reply:
Can you add the silent option to the existing test:coverage script?

Then, this horse is dead.

Reply:
I forgot the script was already there haha. OK, sounds good. 🐴